frontend/src/hooks/useLoadPostList.tsx

```diff
@@ -6,12 +6,17 @@ import { useBlogStore } from "~/store";
 
 /** Listen for BlogErrors and trigger toasts to notify the user about them. */
 export const useLoadPostList = (loadFullList: boolean = false) => {
-  const { blog, publicView } = useBlog();
+  const { blog } = useBlog();
   const setPostPageSize = useBlogStore((state) => state.setPostPageSize);
   // Load all posts with recurcive fetchNextPage calls.
   const {
     query: { fetchNextPage, hasNextPage, isSuccess, data },
-  } = usePostsList(blog?._id, PostState.PUBLISHED, false, !publicView);
+  } = usePostsList(
```

Comment:
@Romu-C ce serait peut être mieux d'utiliser un objet pour passer les params , plutôt que de se souvenir de l'ordre :D 
Et je ne vois pas la désactivation de la query avec `enabled` , t'es sur que la requête part pas quand même ?

Reply:
Oui c'est vrai que ça sera plus clair avec un objet je change ça.
Pour l'appel c'est sur que ça le fait pas vu qu'il est fait uniquement si le param `withViews` est `true`
https://github.com/edificeio/blog/blob/chore/WB-3217_unnecessary-call-for-audience/frontend/src/services/queries/blog.tsx#L249